lib/pure/parsesql.nim

```diff
@@ -566,18 +573,36 @@ type
   SqlParser* = object of SqlLexer ## SQL parser object
     tok: Token
 
+
 {.deprecated: [EInvalidSql: SqlParseError, PSqlNode: SqlNode,
     TSqlNode: SqlNodeObj, TSqlParser: SqlParser, TSqlNodeKind: SqlNodeKind].}
 
-proc newNode(k: SqlNodeKind): SqlNode =
+proc repr*(node: SqlNode): string =
+  result = $node.kind
+  if node.kind in LiteralNodes:
+    result &= " \"" & node.strVal & "\""
```

Comment:
That is not how to quote things.

Reply:
I don't like how this `repr` works anyways. I'll just remove the whole function for now.